math/modular_inverse_simple.cpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file
+ * @brief simple implementation of adaption modular multiplicative inverse
+ * algorithm
+ *
+ * @details
+ * this algorithm calculate the modular inverse iteratively
+ *
+ * @ref
+ * https://en.wikipedia.org/wiki/Modular_multiplicative_inverse
+ */
+
+#include <cassert>
+#include <iostream>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```